pype_schema/parse_json.py

```diff
@@ -911,10 +1008,9 @@ def tag_to_dict(tag_obj):
         """
         tag_dict = {}
         if isinstance(tag_obj, VirtualTag):
+            tag_dict["units"] = "{!s}".format(tag_obj.units)
             tag_dict["tags"] = [tag.id for tag in tag_obj.tags]
-            tag_dict["unary_operations"] = tag_obj.unary_operations
-            tag_dict["binary_operations"] = tag_obj.binary_operations
-            tag_dict["parent_id"] = tag_obj.parent_id
```

Comment:
I'd like to chat briefly about `parent_id` and how it's handled in your updated code just to make sure I understand it